art/test.py

```diff
@@ -3591,6 +3591,83 @@
 <BLANKLINE>
 <BLANKLINE>
 <BLANKLINE>
+>>> tprint("test1\ntest2\ntest3")
+ _               _    _
+| |_   ___  ___ | |_ / |
+| __| / _ \/ __|| __|| |
+| |_ |  __/\__ \| |_ | |
+ \__| \___||___/ \__||_|
+<BLANKLINE>
+ _               _    ____
+| |_   ___  ___ | |_ |___ \
+| __| / _ \/ __|| __|  __) |
+| |_ |  __/\__ \| |_  / __/
+ \__| \___||___/ \__||_____|
+<BLANKLINE>
+ _               _    _____
+| |_   ___  ___ | |_ |___ /
+| __| / _ \/ __|| __|  |_ \
+| |_ |  __/\__ \| |_  ___) |
+ \__| \___||___/ \__||____/
+<BLANKLINE>
+<BLANKLINE>
+>>> tprint("\n\ntest1\ntest2\ntest3")
+<BLANKLINE>
+<BLANKLINE>
+ _               _    _
+| |_   ___  ___ | |_ / |
+| __| / _ \/ __|| __|| |
+| |_ |  __/\__ \| |_ | |
+ \__| \___||___/ \__||_|
+<BLANKLINE>
+ _               _    ____
+| |_   ___  ___ | |_ |___ \
+| __| / _ \/ __|| __|  __) |
+| |_ |  __/\__ \| |_  / __/
+ \__| \___||___/ \__||_____|
+<BLANKLINE>
+ _               _    _____
+| |_   ___  ___ | |_ |___ /
+| __| / _ \/ __|| __|  |_ \
+| |_ |  __/\__ \| |_  ___) |
+ \__| \___||___/ \__||____/
+<BLANKLINE>
+<BLANKLINE>
+>>> tprint("\n\ntest1\ntest2\ntest3\n\n\n",font="lcd")
+<BLANKLINE>
+<BLANKLINE>
+                         _
+  |                 |     |
+ -+-   -       -   -+-    +
+  |   |/       \    |     |
+   -   --      -     -   ---
+<BLANKLINE>
+                         ___
+  |                 |       |
+ -+-   -       -   -+-   -+-
+  |   |/       \    |   |
+   -   --      -     -   ---
+<BLANKLINE>
+                         ___
+  |                 |       |
+ -+-   -       -   -+-   -+-
+  |   |/       \    |       |
+   -   --      -     -   ---
+<BLANKLINE>
+<BLANKLINE>
+<BLANKLINE>
+<BLANKLINE>
+<BLANKLINE>
+>>> tprint("",font="lcd")
+<BLANKLINE>
```

Comment:
From last comment i meant this test, it should be more logical if it prints "".

Reply:
You are right, this test case of `tprint` function is a little vague, so I add more tests for `text2art` --> https://github.com/sepandhaghighi/art/pull/114/commits/40017c3344d297d116d87c04738f8ab21806dae5
And also take a look at this example  : 

```pycon
>>> print("")

>>> s = ""
>>> print(s)

>>>

```

Reply:
Yes, you are right, it should matches with `print`.